### ISSUE TITLE
Filter empty balances

### ISSUE THREAD

**Issue by tbm** [redstreet/fava_miler#7]
fava_miler shows entries for miles where I have a 0 balance. I think they should be filtered out.

**Comment by redstreet** [comment]
Good idea. I'll try to get to it at some point. Patch welcome meanwhile.
